@xen-orchestra/web-core/lib/components/menu/MenuList.vue

```diff
@@ -1,11 +1,20 @@
 <!-- v1.0 -->
 <template>
-  <slot :is-open="isOpen" :open="open" name="trigger" />
-  <Teleport :disabled="!shouldTeleport" to="body">
-    <ul v-if="!hasTrigger || isOpen" ref="menu" :class="{ horizontal, shadow }" class="menu-list" v-bind="$attrs">
+  <slot :is-open="isOpen" :open="open" name="trigger" :is-header="false" />
+  <MobileFullscreen :disabled="!hasTrigger" :isOpen="isOpen" @close="isOpen = false">
```

Comment:
The prop name should be in kebab-case:
```suggestion
  <MobileFullscreen :disabled="!hasTrigger" :is-open="isOpen" @close="isOpen = false">
```